bin/lib/zopen-build

```diff
@@ -305,27 +310,38 @@ checkEnv()
     if [ "${ZOPEN_TARBALL_DEPS}x" = "x" ]; then
       printError "ZOPEN_DEPS or ZOPEN_TARBALL_DEPS needs to be defined to the ported tools this depends on"
     fi
-    deps="${ZOPEN_TARBALL_DEPS}"
+    implicitDeps="curl tar"
+    ext=${ZOPEN_TARBALL_URL##*.}
+    if [ "${ext}x" = "xzx" ]; then
+      implicitDeps="${implicitDeps} xz"
+    elif [ "${ext}x" = "gzx" ]; then
+      implicitDeps="${implicitDeps} gzip"
+    elif [ "${ext}x" = "bz2x" ]; then
+      implicitDeps="${implicitDeps} bzip2"
+    fi
+    printVerbose "Implicitly adding dependencies: ${implicitDeps}"
+    ZOPEN_TARBALL_DEPS="${implicitDeps} ${ZOPEN_TARBALL_DEPS}"
   elif [ "${ZOPEN_TYPE}x" = "GITx" ]; then
     if [ "${ZOPEN_GIT_URL}x" = "x" ]; then
```

Comment:
The check doesn't match the message.  If ZOPEN_GIT_URL is undefined but ZOPEN_URL is, this message will be displayed..

Reply:
I think line 284 should cover it:
```
  elif [ "${ZOPEN_TYPE}x" = "GITx" ]; then
    if [ "${ZOPEN_GIT_URL}x" = "x" ]; then
      export ZOPEN_GIT_URL="${ZOPEN_URL}"
    fi
```